--- /snap/lxd/current/lxcfs/lxc.mount.hook	2022-03-13 17:39:48.000000000 +0000
+++ lxc.mount.hook	2022-03-14 14:52:09.311244584 +0000
@@ -37,7 +37,9 @@ if [ -d /var/snap/lxd/common/var/lib/lxc
 fi
 
 # Allow nesting lxcfs
-if [ -d "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs/" ]; then
+if [ -d "${LXC_ROOTFS_MOUNT}/var/snap/lxd/" ]; then
+    rm -Rf "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs"
+    mkdir -p "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs"
     mount -n --bind /var/snap/lxd/common/var/lib/lxcfs "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs/"
 fi
 
